xml/qemu_overview.xml

```diff
@@ -91,6 +83,11 @@
     <command>qemu-system-x86_64</command>
    </para>
   </listitem>
+  <listitem>
+   <para>
+    <command>qemu-system-arm</command>
+   </para>
+  </listitem>
```

Comment:
we only provide the matching qemu arch CPU type on SLE. Which means: only qemu-system-x86_64 on x86_64 etc.... so this should be removed.